lib/src/response.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:dio/dio.dart';
+import 'package:http_mock_adapter/src/interfaces.dart';
+import 'dart:typed_data';
+import 'dart:convert';
+
+/// wraper of  [Dio]'s ResponseBody
```

Comment:
wrapper of [Dio]* (typo and extra space)

Reply:
okay